README.md

```diff
@@ -165,5 +196,11 @@ relationships between the two are set up in Active Record.
 
 Write your code here:
 ```ruby
-# code here
+1. lamp = Lamp.create(wishes_remaining: 3), genie = Genie.create(name: "Genie")
+2. genie.lamp = lamp
```

Comment:
need to call `.save` later if you want this change to persist, or use `.update` to set the value and save in one command